src/plugins/explore/public/application/legacy/discover/application/components/chart/hits_counter/hits_counter.tsx

```diff
@@ -69,38 +79,25 @@ export function HitsCounter({ hits, showResetButton, onResetQuery, rows }: HitsC
       >
         <EuiFlexItem grow={false}>
           <EuiText size="s">
-            <strong>
+            {hits && (
```

Comment:
if we don't have hit count, then none of this shows up? For esxample, in SQL, we don't have hits count. I think we should account for the scenarios when we don't have hits count